roles/pulp3-postgresql/README.md

```diff
@@ -10,19 +10,43 @@ More specifically, this role does the following:
 2. Install the package needed to talk to PostgreSQL from Python, if any.
 3. Create and run migrations for `pulp_app`.
 
-Variables:
+Role Variables:
+---------------
 
-* `pulp_postgresql_call_anxs_postgresql`: Optional, defaults to true. Whether to
-  call [anxs.postgresql](https://galaxy.ansible.com/anxs/postgresql).
-* `pulp_database_config`: Optional. Defines how Pulp will talk to PostgreSQL.
-* `pulp_config_dir`, `pulp_install_dir`, `pulp_user`: Optional. The meaning is
-  the same as for the `pulp3` role.
+* `pulp_postgresql_call_anxs_postgresql`: Defaults to true. Whether to call
+  [anxs.postgresql](https://galaxy.ansible.com/anxs/postgresql).
+* `pulp_database_config`: Defines how Pulp will talk to PostgreSQL. Defaults
```

Comment:
How do I specify this? Our docs[0] just point to django docs that have python configuration.  How do i express this for ansible? Perhaps it would be better if I could specify a settings.py file that should be used.

https://docs.pulpproject.org/en/3.0/nightly/installation/configuration.html#databases